src/components/ImageThumbnail/index.js

```diff
@@ -55,6 +57,9 @@ class ImageThumbnail extends React.PureComponent {
         let classname = this.props.selected ? 'image-thumb selected' : 'image-thumb'
         const bgStyle = {backgroundImage: 'url(' + this.props.url + ')'};
         let editModal = null;
+        const {user: {userType}, isSharedImage} = this.props;
```

Comment:
On the main event form, opening the "Choose from defaults" image selection causes a crash due to user being null.

Reply:
Fixed.